crawler/dockercontainer.py

```diff
@@ -270,9 +285,9 @@ def _parse_log_locations(self, var=None):
         data = None
         try:
             data = misc.GetProcessEnv(container.pid)[var]
-	    log_locations = data.split(',')
-	    logs = [{'name': name.strip(), 'type': None}
-		    for name in log_locations]
+            log_locations = data.split(',')
+            logs = [{'name': name.strip(), 'type': None}
```

Comment:
This is more pythonic, also pylint measures number of local variables as a quality measure (flags warnings if number exceeds say 15).

 logs = [{'name': name.strip(), 'type': None}
                    for name in  misc.GetProcessEnv(container.pid)[var].split(',')]